tests/test_postproc.py

```diff
@@ -0,0 +1,56 @@
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+"""
+tests.test_postproc- Tests of various functions in newspack, among which rar_renamer()
+"""
+
+from sabnzbd.postproc import *
+from unittest import mock
+import os, shutil
+
+
+class TestPostProc:
+
+    # Tests of rar_renamer() (=deobfuscate) against various input directories
+    def test_rar_renamer(self):
+
+        # Function to deobfuscate one directorty with rar_renamer()
+        def deobfuscate_dir(sourcedir):
+            # sourcedir is the relative path to the directory with obfuscated files
+
+            # enrich to absolute path:
+            sourcedir = os.path.join(os.getcwd(), sourcedir)
+            # We create a workingdir, because the filenames are really changed
+            workingdir = sourcedir + "---WORKING-DIR"
+            # if workingdir still there from previous run, remove it:
+            try:
+                shutil.rmtree(workingdir)
+            except:
+                pass
+            # create a fresh copy
+            try:
+                shutil.copytree(sourcedir, workingdir)
+            except:
+                assert False
```

Comment:
Pytest has some nice tools for this to make the errors more clear. You can say this:
```python
pytest.fail("Could not create copy of files for rar_renamer")
```
This way it will display a better error instead of just an assertion-failure.

Reply:
:+1: